envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/permissions/IncomingPermissionsLoggingModeTest.kt

```diff
@@ -657,6 +658,32 @@ internal class IncomingPermissionsLoggingModeTest : EnvoyControlTestConfiguratio
         )
     }
 
+    @Test
```

Comment:
Looks like a candidate for the extensions approach. However, I understand that rewriting the whole suite isn't feasible in this PR.

Reply:
I know, I am just really want to avoid rewriting this test in this PR :)